qcommon/src/main/java/jake2/qcommon/network/netchan_t.java

```diff
@@ -57,13 +60,10 @@
 
     public int last_reliable_sequence; // sequence number of last send
 
-    //	   reliable staging and holding areas
-    public sizebuf_t reliable = new sizebuf_t(); // writing buffer to send to
-                                                // server
-
-    public byte reliable_data[] = new byte[Defines.MAX_MSGLEN - 16]; // leave
-                                                                   // space for
-                                                                   // header
+    // reliable staging and holding areas
+    // writing buffer to send to server
+    // max size: Defines.MAX_MSGLEN - 16: leave space for header
```

Comment:
The "size" here  is somewhat confusing.
When I read it I understood it as "number of elements" or "length" but when I look at the code that was replaced with this I think that it refers to the number of bytes in serialized representation of these messages

Reply:
Exactly. Will move this comment to the place that is more related to bytes

Reply:
updated